src/main/resources/edu/cornell/kfs/coa/document/datadictionary/ReversionCategoryMaintenanceDocument.xml

```diff
@@ -1,62 +1,47 @@
+        </property>
+    </bean>
 
-  <bean id="ReversionCategoryMaintenanceDocument-parentBean" abstract="true" parent="MaintenanceDocumentEntry">
-    <property name="businessObjectClass" value="edu.cornell.kfs.coa.businessobject.ReversionCategory"/>
-    <property name="maintainableClass" value="edu.cornell.kfs.coa.document.ReversionCategoryMaintainableImpl"/>
-    <property name="maintainableSections">
-      <list>
-        <ref bean="ReversionCategoryMaintenanceDocument-EditReversionCategory"/>
-      </list>
-    </property>
-    <property name="lockingKeys">
-      <list>
-        <value>reversionCategoryCode</value>
-      </list>
-    </property>
-    <property name="documentTypeName" value="ORGC"/>
-    <property name="documentAuthorizerClass" value="org.kuali.kfs.kns.document.authorization.MaintenanceDocumentAuthorizerBase"/>
-    <property name="workflowAttributes">
-    	<ref bean="ReversionCategoryMaintenanceDocument-workflowAttributes"/>
-    </property>
-  </bean>
 
-<!-- Maintenance Section Definitions -->
+    <!-- Maintenance Section Definitions -->
 
-  <bean id="ReversionCategoryMaintenanceDocument-EditReversionCategory" parent="ReversionCategoryMaintenanceDocument-EditReversionCategory-parentBean"/>
-
-  <bean id="ReversionCategoryMaintenanceDocument-EditReversionCategory-parentBean" abstract="true" parent="MaintainableSectionDefinition">
-    <property name="id" value="Edit Reversion Category"/>
-    <property name="title" value="Edit Reversion Category"/>
-    <property name="maintainableItems">
-      <list>
-        <bean parent="MaintainableFieldDefinition" p:name="reversionCategoryCode" p:required="true"/>
-        <bean parent="MaintainableFieldDefinition" p:name="reversionCategoryName" p:required="true"/>
-        <bean parent="MaintainableFieldDefinition" p:name="reversionSortCode" p:required="true"/>
-        <bean parent="MaintainableFieldDefinition" p:name="active" p:defaultValue="true" p:required="true"/>
-      </list>
-    </property>
-  </bean>
+    <bean id="ReversionCategoryMaintenanceDocument-EditReversionCategory" parent="ReversionCategoryMaintenanceDocument-EditReversionCategory-parentBean"/>
+    <bean id="ReversionCategoryMaintenanceDocument-EditReversionCategory-parentBean" abstract="true" parent="MaintainableSectionDefinition"
+          p:id="Edit Reversion Category"
+          p:title="Edit Reversion Category">
+        <property name="maintainableItems">
+            <list>
+                <bean parent="MaintainableFieldDefinition" p:name="reversionCategoryCode" p:required="true" p:newLookup="true"/>
+                <bean parent="MaintainableFieldDefinition" p:name="reversionCategoryName" p:required="true" p:newLookup="true"/>
+                <bean parent="MaintainableFieldDefinition" p:name="reversionSortCode" p:required="true" p:newLookup="true"/>
+                <bean parent="MaintainableFieldDefinition" p:name="active" p:defaultValue="true" p:required="true" p:newLookup="true"/>
```

Comment:
Can these fields omit the `newLookup` setting? It doesn't look like these four fields are linking to any other BOs.